Python/bytecodes.c

```diff
@@ -507,12 +507,12 @@ dummy_func(
             DISPATCH_INLINED(new_frame);
         }
 
-        inst(LIST_APPEND, (list, unused[oparg-1], v -- list : PyList_Type, unused[oparg-1])) {
+        inst(LIST_APPEND, (list, unused[oparg-1], v -- list : <<= PyList_Type, unused[oparg-1])) {
             ERROR_IF(_PyList_AppendTakeRef((PyListObject *)list, v) < 0, error);
             PREDICT(JUMP_BACKWARD);
         }
 
-        inst(SET_ADD, (set, unused[oparg-1], v -- set, unused[oparg-1])) {
+        inst(SET_ADD, (set, unused[oparg-1], v -- set : <<= PySet_Type, unused[oparg-1])) {
             int err = PySet_Add(set, v);
```

Comment:
Note to self: these are on a new type because they are opcodes for implementing list and set comprehensions.

See https://docs.python.org/3/library/dis.html for information.